content/en/03-using-pixie/02-using-cli.md

```diff
@@ -142,7 +144,23 @@ px run -f <script.pxl>		# Use Pixie's CLI to run the pxl script with the provide
 px live -f <script.pxl>		# Use Pixie's Live CLI to run the pxl script with the provided filename
 ```
 
-## Debugging Pixie using the CLI
+## Advanced
+
+### Disabling Encryption
+
+By default, Pixie implements end-to-end encryption for telemetry data requested by the CLI. For more information, see the [FAQ](/about-pixie/faq#how-does-pixie-secure-its-data).
+
+For faster script execution, opt out of E2E encryption for a specific query using the `--disable_e2e_encryption` flag:
```

Comment:
Use the `--e2e_encryption` flag to control encryption of table data. Enabling E2E encryption adds some overhead to encrypt the results on the Vizier side and decrypt it on the CLI side.